src/rawtoaces_util/image_converter.cpp

```diff
@@ -359,10 +370,10 @@ bool prepare_transform_DNG(
     std::vector<std::vector<double>> &IDT_matrix,
     std::vector<std::vector<double>> &CAT_matrix )
 {
-    std::string camera_make  = settings.custom_camera_make;
-    std::string camera_model = settings.custom_camera_model;
-    if ( !fetch_camera_make_and_model( image_spec, camera_make, camera_model ) )
-        return false;
+    // TO Does this function even need the camera info?
+    // core::CameraInfo camera_info = get_camera_info( image_spec, settings );
+    // if ( camera_info.is_empty() )
+    //     return false;
```

Comment:
was it used just for validation `settings` that it has camera settings? because camera_make wasn't used anywhere